code/core/src/components/components/tooltip/InteractiveTooltipWrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo } from 'react';
+
+import { styled } from 'storybook/theming';
+
+import { shortcutToHumanString } from '../../../manager-api';
+import type { API_KeyCollection } from '../../../manager-api/modules/shortcuts';
+import { TooltipNote } from './TooltipNote';
+import { WithTooltip } from './WithTooltip';
+
+const NoMarginNote = styled(TooltipNote)(() => ({
+  margin: 0,
+}));
+
+// TODO: Improve delay management; make the delay near-instantaneous if any instance of this component has been recently shown.
+// TODO: Find way to trigger the tooltip when the child component is focused too. Will need this for general a11y but particularly for Sidebar nav secondary actions.
```

Comment:
@ghengeveld what's the policy with JSDoc in code? Should I make issues? Leave these comments here? Remove them and wait for the issues to surface from user reports?